shell/platform/windows/flutter_windows_engine.h

```diff
@@ -190,6 +190,9 @@ class FlutterWindowsEngine {
   // given |texture_id|.
   bool MarkExternalTextureFrameAvailable(int64_t texture_id);
 
+  // Posts the given callback onto the raster thread.
+  bool PostRasterThreadTask(std::function<void()> callback);
```

Comment:
Consider using `fml::closure` instead of `std::function<void()>`.

```suggestion
  bool PostRasterThreadTask(fml::closure callback);
```